Description: Fix type mismatch
 This patch fixes the type mismatch happening because of the newer indicatif crate.
 Thanks to bnoctis for pointing me in the right direction.
Author: Matthias Geiger <matthias.geiger1024@tutanota.de>
Forwarded: yes
Last-Update: 2023-02-02
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/debian.rs
+++ b/src/debian.rs
@@ -115,7 +115,7 @@
     let pb = ProgressBar::new(jobs as u64)
         .with_style(
             ProgressStyle::default_bar()
-                .template("[{pos:.green}/{len:.green}] {prefix:.bold} {wide_bar}"),
+                .template("[{pos:.green}/{len:.green}] {prefix:.bold} {wide_bar}")?,
         )
         .with_prefix("Resolving debian packages");
     pb.tick();
